app/models/budget.rb

```diff
@@ -76,6 +77,10 @@ def any_update_audits?
     audit_version.nil?
   end
 
+  private def value_changed
```

Comment:
maybe we should call this `validate_value_changed` to distinguish it from the ActiveRecord-generated `value_changed?` method?

Reply:
`ensure` rather than `validate`?